subsys/net/ip/Kconfig

```diff
@@ -640,6 +648,18 @@ config NET_PKT_TXTIME
 	  when the application wants to set the exact time when the network
 	  packet should be sent.
 
+config NET_PKT_RXTIME_STATS
+	bool "Enable network packet RX time statistics"
+	select NET_PKT_TIMESTAMP
+	select NET_STATISTICS
+	depends on (NET_UDP || NET_TCP)
+	help
+	  Enable network packet RX time statistics support. This can be used to
```

Comment:
Also no clear mentioning that the statistics is cumulative for all packets received.